packages/playwright-core/src/server/chromium/crPage.ts

```diff
@@ -432,6 +432,7 @@ class FrameSession {
       this._firstNonInitialNavigationCommittedFulfill = f;
       this._firstNonInitialNavigationCommittedReject = r;
     });
+    this._firstNonInitialNavigationCommittedPromise.catch(() => {});
```

Comment:
Maybe do it inside `FrameSession._initialize` and add a comment there that we may get an exception before awaiting the promise?

Reply:
added a comment. I think having it close to where the Promise is created is also reasonable. Similar to what we do with `this._stopVideoRecording().catch(() => {});`.